dist/en/integrations/shopify/shopify_2022-02_to_shopify_2024-10.txt

```diff
@@ -1,4 +1,4 @@
-# Shopify Product Taxonomy - Mapping shopify/2022-02 to shopify/2024-07
+# Shopify Product Taxonomy - Mapping shopify/2022-02 to shopify/2024-10
```

Comment:
Now these ones though — should this not be `2024-07 to 2024-10-unstable`? cc @jeanrick

Reply:
Yea, if we're including the `unstable` labels in `dist` then this version should be `2024-10-unstable`. The file name will also change from `shopify_2022-02_to_shopify_2024-07.txt` to `shopify_2022-02_to_shopify_2024-10-unstable.txt`

Reply:
An alternative would be reserve `unstable` only in file contents but removing the `unstable` from file names.   Then when we cut a new release e.g., from `2024-10-unstable` to `2024-10`,  the file names would be unchanged instead of replacing by new files where the diff would include the whole file contents.   If the file names are unchanged, all diff in the release PR are about version value changes in files.  

If that makes sense to you, I'll implement it. cc @jeanrick @elsom25

Reply:
we can add the following code after https://github.com/Shopify/product-taxonomy/blob/e9d8dbe5cd0f6c6eb7914bb0482ba9e9acc781a3/app/commands/generate_dist_command.rb#L115-L116
```
    if input_version.include?("-unstable")
      input_version = input_version.delete_suffix("-unstable")
    end

    if output_version.include?("-unstable")
      output_version = output_version.delete_suffix("-unstable")
    end
```

Then https://github.com/Shopify/product-taxonomy/blob/e9d8dbe5cd0f6c6eb7914bb0482ba9e9acc781a3/app/commands/generate_dist_command.rb#L119-L120  will not include "unstable" to mapping file names

Reply:
I can see the merit of clearly saying "these files are unstable" via the file name, but I'm not a fan of the whole file getting replaced when we cut the release 🤔 

But given that we don't do this for the other taxonomy files, I lean to keeping it out of the filename for now